webapp/src/modules/transaction/sagas.js

```diff
@@ -36,7 +36,8 @@ function* handleTransactionRequest(action = {}) {
     yield put(
       fetchTransactionSuccess({
         ...transaction,
-        status: TRANSACTION_STATUS.confirmed
+        status: TRANSACTION_STATUS.confirmed,
+        receipt
```

Comment:
note to self: we need to add this to `decentraland-dapps`

Reply:
@cazala  Do you want me to create an issue?

Reply:
https://github.com/decentraland/decentraland-dapps/issues/8